core/spec/models/spree/product_spec.rb

```diff
@@ -244,9 +244,28 @@ def duplicate_extra(old_product)
 
         expect(product2.slug).to eq 'test-456'
       end
-
     end
 
+    context 'history' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.